src/stars.scss

```
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
```

+ add styles for `.stars`
```suggestion
&__star {
```